### ISSUE TITLE
 Packit Onboarding Guide TOC

### ISSUE THREAD

**Issue by pemensik** [packit/packit.dev#665]
I were looking for table of contents in the onboarding guide. Who does it expect on the right side? I certainly did not, because I kind of expect advertisements there. It seems to me there should be ToC at least for GitHub, Gitlab difference. With images this page seems too long. It contains too many header levels. Github has 3 subsections in section 1. at the same level as GitLab.

I would recommend splitting onboarding guide into more separate pages. ToC should be in the center below the main title, maybe with button to expand also subsections. The page should have visually different h2, h3 and h4. They are too similar IMO, fast scrolling does not hint where exactly github ends.

I searched for information how to integrate with dist-git only, because I am not upstream for most of my packages and have no direct commit rights. Unfortunately I haven't found any trace of such feature, which I heard is supported from Mirek. 

I have found some mention at:
https://packit.dev/docs/fedora-releases-guide/#pull-from-upstream-job

But I would expect it more straightforward. Maybe onboarding guide should mention early what kind of access do I need in upstream repository used. It seems to me it still requires admin access to repository and does not mention it explicitly before it starts describing steps, which I cannot do without it. If it can do something useful without that, I think it should say so before describing steps required on each forge.

**Comment by lachmanfrantisek** [comment]
Hi @pemensik !

Thanks for the suggestions!

I read that as two tasks:
* [ ] Better structure the page and TOP
* [ ] Describe differences between `pull-from-upstream` and `propose-downstream` at beginning.

(Regarding the split to multiple pages, we've already had that and it wasn't ideal either.)

Thanks again for the suggestion!

**Comment by nforro** [comment]
I think we could have two major sections of the page (Fedora releases guide):

- I'm an upstream maintainer and I want my releases to be automatically packaged for Fedora
- I'm a downstream maintainer and I want to consume upstream releases automatically

And each would describe which job to use and what prerequisites are needed.

**Comment by pemensik** [comment]
Just a detail: I have noticed there actually is ToC on onboarding page only after I have opened an issue here. I wanted to report there is one missing and there should be. Instead I found it only after that, which suggest UX should be improved.

> And each would describe which job to use and what prerequisites are needed.

That is exactly what I were looking for and failed to find. I think that is the most important information before I start.
